cli/tests/fixtures/extensions/permissions/correct-read-perms/main.ts

```diff
@@ -0,0 +1,4 @@
+import { walk, exists } from "https://deno.land/std@0.143.0/fs/mod.ts"
```

Comment:
I don't see **any** reason why we'd have separate test cases for correct/incorrect. Can't we just merge all of these together? The only difference should be the permissions, not the actual test extension. Anything else is just confusing.

Reply:
I agree it feels a bit unwieldy to have many fixtures; but these are integration tests, and I want to make sure to test the whole process of installing and running that behaves as expected. If I were to change the permissions in-flight, I could have parts of the code uncovered.

For example, someone could go touch the mechanism for deserializing the manifest (could be as simple as adding `#[serde(rename)]`) and no test would break if we are building the permissions manually.